packages/notification-service/src/blockscout/transfers.ts

```diff
@@ -57,26 +59,31 @@ async function getLatestTokenTransfers(
   }
 
   if (!response.result.length) {
-    console.debug('No new logs found for token:', tokenAddress)
+    console.debug('No new logs found for token:', currency)
     return { transfers: null, latestBlock: lastBlockNotified }
   }
 
-  console.debug('New logs found for token:', tokenAddress, response.result.length)
+  console.debug('New logs found for token:', currency, response.result.length)
   const { transfers, latestBlock } = formatTransfers(response.result as TokenTransfer[], currency)
   return { transfers, latestBlock }
 }
 
 export function filterAndJoinTransfers(
   celoTransfersByTx: Map<string, Transfer[]> | null,
-  stableTransfersByTx: Map<string, Transfer[]> | null
+  cUsdTransfersByTx: Map<string, Transfer[]> | null,
+  cEurTransfersByTx: Map<string, Transfer[]> | null
 ): Transfer[] {
-  const stableTransfers = stableTransfersByTx ? flat([...stableTransfersByTx.values()]) : []
+  const cUsdTransfers = cUsdTransfersByTx ? flat([...cUsdTransfersByTx.values()]) : []
+  const cEurTransfers = cEurTransfersByTx ? flat([...cEurTransfersByTx.values()]) : []
   const celoTransfers = celoTransfersByTx ? flat([...celoTransfersByTx.values()]) : []
 
   // Exclude transaction found in both maps as those are from exchanges
-  const filteredCelo = celoTransfers.filter((t) => !stableTransfersByTx?.has(t.txHash))
-  const filteredStable = stableTransfers.filter((t) => !celoTransfersByTx?.has(t.txHash))
-  return filteredCelo.concat(filteredStable)
+  const filteredCelo = celoTransfers.filter(
+    (t) => !cUsdTransfersByTx?.has(t.txHash) && !cEurTransfersByTx?.has(t.txHash)
+  )
+  const filteredCusd = cUsdTransfers.filter((t) => !celoTransfersByTx?.has(t.txHash))
+  const filteredCEur = cEurTransfers.filter((t) => !celoTransfersByTx?.has(t.txHash))
+  return filteredCelo.concat(filteredCusd).concat(filteredCEur)
```

Comment:
nit: `Array.concat` supports N args
```suggestion
  return filteredCelo.concat(filteredCusd, filteredCEur)
```